rendering/benchmark.js

```diff
@@ -0,0 +1,317 @@
+#! /usr/bin/env node
+const puppeteer = require('puppeteer');
+const webpack = require('webpack');
+const config = require('./webpack.config');
+const webpackMiddleware = require('webpack-dev-middleware');
+const http = require('http');
+const path = require('path');
+const fs = require('fs');
+const yargs = require('yargs');
+const log = require('loglevelnext');
+const serveStatic = require('serve-static');
+
+const compiler = webpack(Object.assign({mode: 'development'}, config));
+
+function getHref(entry) {
+  return path.dirname(entry).slice(1) + '/';
+}
+
+const staticHandler = serveStatic(__dirname, {
+  'setHeaders': (res, path) => {
+    // add the correct headers for pbf tiles on disk
+    if (path.match(/\.pbf$/)) {
+      res.setHeader('Content-Type', 'application/x-protobuf');
+      res.setHeader('Content-Encoding', 'gzip');
+    }
+  }
+});
+
+const defaultHandler = serveStatic(path.join(__dirname, 'default'));
+
+function indexHandler(req, res) {
+  const items = [];
+  for (const key in config.entry) {
+    const href = getHref(config.entry[key]);
+    items.push(`<li><a href="${href}">${href}</a></li>`);
+  }
+  const markup = `<!DOCTYPE html><body><ul>${items.join('')}</ul></body>`;
+
+  res.writeHead(404, {
```

Comment:
```suggestion
  res.writeHead(200, {
```

Reply:
This logic was copied from here: https://github.com/openlayers/openlayers/blob/a9b2952be4ef204db9e7c52cea9cdc92b7489c6e/rendering/test.js#L35-L38

Although I agree this could probably be mutualized